packages/react-jsx-highcharts/src/components/Tooltip/Tooltip.js

```diff
@@ -36,8 +35,9 @@ const Tooltip = memo(props => {
 });
 
 const updateTooltip = (chart, config) => {
-  const tooltip = chart.object.tooltip;
-  tooltip.update(config);
+  chart.update({
```

Comment:
May need to check that this, doesn't undo https://github.com/whawker/react-jsx-highcharts/issues/226#issuecomment-529985171

Reply:
Sorry, just noticed you already referenced this bug on #289 - apologies